modules/spacecraftinstruments/rendering/renderablecrawlingline.cpp

```diff
@@ -50,7 +50,8 @@ namespace {
 } // namespace
 
 // @TODO:  This clas is not properly working anymore and needs to be substantially
```

Comment:
Since you are changing this comment anyways, you could change the spelling to "This class"
(Sorry for spelling OCD)